test/images/node-perf/tf-wide-deep/Dockerfile

```diff
@@ -14,8 +14,15 @@
 
 FROM BASEIMAGE
 
+CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
+
 RUN apt-get update && apt-get install -y wget time
 RUN pip install tensorflow
```

Comment:
you need to remove this line as you are adding following if/else block for individual archs.

Reply:
@mkumatag 
Hi,
In my opinion, we are not register the 'F' flag for 'binfmt_misc', and so we need `qemu-static` inside the container.
If without this line. The building will be failed.
Please see following as reference:

make all WHAT=node-perf/tf-wide-deep
...
.,.
Step 1/7 : FROM arm64v8/python:3.6-slim-stretch
3.6-slim-stretch: Pulling from arm64v8/python
Digest: sha256:bd9b8e8f25e7eafdd1c09583d57bb0362645ea294f01c714b88f486931a9cf45
Status: Image is up to date for arm64v8/python:3.6-slim-stretch
 ---> 8e3c7c26d38b
Step 2/7 : RUN apt-get update && apt-get install -y wget time
 ---> Running in afe74ccd5e2d
standard_init_linux.go:190: exec user process caused "no such file or directory"
The command '/bin/sh -c apt-get update && apt-get install -y wget time' returned a non-zero code: 1
Makefile:37: recipe for target 'all-container' failed
make: *** [all-container] Error 1

Reply:
I'm talking about `RUN pip install tensorflow` line

Reply:
@mkumatag 
Thank you very much.
I have removed it. Please check.